rollup.config.js

```diff
@@ -1,46 +1,45 @@
-import typescript from "@rollup/plugin-typescript";
-import json from '@rollup/plugin-json';
-import { importAssertionsPlugin } from 'rollup-plugin-import-assert';
-import { importAssertions } from 'acorn-import-assertions';
+import typescript from '@rollup/plugin-typescript'
+import json from '@rollup/plugin-json'
+import { importAssertionsPlugin } from 'rollup-plugin-import-assert'
+import { importAssertions } from 'acorn-import-assertions'
 import { dts } from 'rollup-plugin-dts'
 
-import pkg from "./package.json";
+import pkg from './package.json' assert { type: 'json' }
 
 export default [
-    // browser-friendly UMD build
-    {
-        input: "src/index.ts",
-        output: {
-            name: "unlockTs",
-            file: pkg.browser,
-            format: "umd",
-        },
-        acornInjectPlugins: [importAssertions],
-
-        plugins: [
-            importAssertionsPlugin(),
-            typescript({ tsconfig: "./tsconfig.json" }),
-            json(),
-        ],
+  // browser-friendly UMD build
+  {
+    input: 'src/index.ts',
+    external: ['axios', 'joi', 'viem', 'viem/chains'],
```

Comment:
added dependenceies here so we can remove the warnings at build. lmk if there is any reason not to include these. see globals references below too